.github/workflows/libvsync-actions.yml

```diff
@@ -0,0 +1,23 @@
+name: libvsync tests
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on: [push]
+jobs:
+  Run-Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
+      - run: echo "🔎 The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
```

Comment:
Could you remove all these testing echos before merging? It makes it hard to see what is actually supposed to be tested in this job.

Reply:
I want to keep them in the initial version. I left them on purpose